js/components/foia_component_card.jsx

```diff
@@ -16,6 +16,8 @@ function Card({ card }) {
       }`}
       href={url}
       onClick={onClick}
+      target={alt ? '_blank' : '_self'}
+      rel={alt ? 'noreferrer' : ''}
```

Comment:
Nit: Better to not set the attribute if we don't have a desired value.

```suggestion
      target={alt ? '_blank' : undefined}
      rel={alt ? 'noreferrer' : undefined}
```